src/config.ts

```diff
@@ -59,9 +59,9 @@ export interface KafkaConfigurations {
 }
 export interface RxNormDbConfig {
 	host: string;
-	user: string;
-	password: string;
-	database: string;
-	port: number;
-	timeout: number;
+	user?: string;
+	password?: string;
+	database?: string;
+	port?: number;
+	connectTimeout?: number;
```

Comment:
I don't think these should be optional

Reply:
They are optional at the `mysql.createPool` level, but making them mandatory in `config.ts` doesn't seem to be a problem

<img width="469" alt="Screenshot 2024-11-11 at 11 17 40 AM" src="https://github.com/user-attachments/assets/c9be3bbd-59c9-4486-97cf-f18fd33b90db">